frontend/src/component/common/ConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/DateSingleValue/DateSingleValue.tsx

```diff
@@ -1,42 +1,88 @@
     setError: React.Dispatch<React.SetStateAction<string>>;
 }
 
+const StyledWrapper = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: theme.spacing(1),
+    alignItems: 'center',
+    gap: theme.spacing(1),
+}));
+
 export const DateSingleValue = ({
     setValue,
     value,
     error,
     setError,
 }: IDateSingleValueProps) => {
+    const timezones = Object.values(
+        TimezoneCountries.getAllTimezones({ deprecated: false })
+    ).map(timezone => ({
+        key: timezone.name,
+        label: `${timezone.name}`,
+        utcOffset: timezone.utcOffsetStr,
+    }));
+    const { timeZone: localTimezoneName } =
+        Intl.DateTimeFormat().resolvedOptions();
+    const [pickedDate, setPickedDate] = useState(value || '');
+
+    const localTimezone = useMemo<ITimezoneSelect | undefined>(() => {
+        return timezones.find(t => t.key === localTimezoneName);
+    }, [timezones, localTimezoneName]);
+
     if (!value) return null;
 
     return (
         <>
             <ConstraintFormHeader>Select a date</ConstraintFormHeader>
-            <Input
-                id="date"
-                label="Date"
-                type="datetime-local"
-                value={parseDateValue(value)}
-                onChange={e => {
-                    setError('');
-                    const parsedDate = parseValidDate(e.target.value);
-                    const dateString = parsedDate?.toISOString();
-                    dateString && setValue(dateString);
-                }}
-                InputLabelProps={{
-                    shrink: true,
-                }}
-                error={Boolean(error)}
-                errorText={error}
-                required
-            />
+            <StyledWrapper>
+                <Input
+                    id="date"
+                    label="Date"
+                    type="datetime-local"
+                    value={parseDateValue(pickedDate)}
+                    onChange={e => {
+                        setError('');
+                        const parsedDate = parseValidDate(e.target.value);
+                        const dateString = parsedDate?.toISOString();
+                        dateString && setPickedDate(dateString);
+                        dateString && setValue(dateString);
+                    }}
+                    InputLabelProps={{
+                        shrink: true,
+                    }}
+                    error={Boolean(error)}
+                    errorText={error}
+                    required
+                />
+                <ConditionallyRender
+                    condition={Boolean(localTimezone)}
+                    show={
+                        <p>
+                            {localTimezone?.key}{' '}
+                            {`(UTC ${localTimezone?.utcOffset})`}
+                        </p>
+                    }
+                    elseShow={<p>No timezone information available</p>}
```

Comment:
So something like this, yeah?

```suggestion
                    elseShow={<p>The time shown is in your local time zone according to your browser.</p>}
```